governance-charter.md

```diff
@@ -63,15 +63,16 @@ can bootstrap the remainder of the process.
 
 The Bootstrap Committee will be replaced by the elected OpenTelemetry Governance
 Committee. Ultimately, the **OpenTelemetry Governance Committee will consist
-of 5 individual members** of the community **elected for 2 year terms**. The
-terms will be **staggered with 1 year elections** (alternating 2 seats and 3
+of 9 individual members** of the community **elected for 2 year terms**. The
```

Comment:
please change last updated on the top of the document

Reply:
(done)